esi_leap/api/controllers/v1/node.py

```diff
@@ -21,8 +21,10 @@
 from esi_leap.api.controllers import types
 from esi_leap.common import ironic
 from esi_leap.common import keystone
+from esi_leap.common import statuses
 import esi_leap.conf
 from esi_leap.objects import offer as offer_obj
+from esi_leap.objects import lease as lease_obj
```

Comment:
I'm not sure why the CI doesn't pick this up, but can you move the lease import above the offer import? 'tox -epep8' complains about imports not being in alphabetical order.

Reply:
Done